app/routes/config/constants.js

```diff
@@ -1,14 +1,14 @@
-'use strict'
+"use strict";
 
 const routes = {
-  profile: '/api/profile',
-  signup: '/api/signup',
-  login: '/api/login',
-  home: '/api/home',
-  googleSignUp: '/auth/google',
-  googleCallback: '/auth/google/callback',
-  posts: '/api/posts'
-}
-
+  profile: "/api/profile",
+  signup: "/api/signup",
+  login: "/api/login",
+  home: "/api/home",
+  googleSignUp: "/auth/google",
+  googleCallback: "/auth/google/callback",
+  posts: "/api/posts",
+  clap: "/api/clap/:post_id"
```

Comment:
we are not creating api by URLS. it will be a post or put request.
When user clicks clap(on front end) - it will trigger a post or put request that updates the clap value in the backend .
Doesnt make sense to use params.